apps/src/JSInterpreter.js

```diff
@@ -73,7 +104,7 @@ var JSInterpreter = module.exports = function (options) {
   };
 
   try {
-    this.interpreter = new window.Interpreter(options.code, initFunc);
+    new window.Interpreter(options.code, initFunc);
```

Comment:
It doesn't appear that we have any linting rule. Comment added in the next PR explaining why I changed this.